polonius-parser/README.MD

```diff
@@ -0,0 +1,78 @@
+# Polonius Parser
```

Comment:
Can you please use `.md` instead of `.MD` as extension?

Reply:
Sure, done.